specification/nginx/Nginx.Management/WafPolicies.tsp

```diff
@@ -0,0 +1,123 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using Rest;
+using Azure.ResourceManager;
+
+namespace Nginx.NginxPlus;
+
+/**
+ * Represents a WAF policy resource under a given Nginx deployment.
+ */
+@parentResource(NginxDeployment)
+model WafPolicy is ProxyResource<NginxDeploymentWafPolicyProperties> {
+  ...ResourceNameParameter<WafPolicy>;
+}
+
+/**
+ * Properties of a WAF policy resource.
+ */
+model NginxDeploymentWafPolicyProperties {
+  /** The raw policy content (base64-encoded). */
+  content: bytes;
+
+  /** The file path of the WAF policy. */
+  filepath: string;
+
+  /** The provisioning state of the WAF policy. */
+  @visibility(Lifecycle.Read)
+  provisioningState?: ProvisioningState;
+
+  /** The compiling status of the WAF policy. */
+  compilingState: NginxDeploymentWafPolicyCompilingStatus;
+
+  /** The applying status of the WAF policy. */
+  applyingState: NginxDeploymentWafPolicyApplyingStatus;
+}
+
+/**
+ * Compilation status for a WAF policy.
+ */
+model NginxDeploymentWafPolicyCompilingStatus {
+  /** The compilation status code. */
+  code: string;
+
+  /** The human-readable compilation status. */
+  displayStatus: string;
+
+  /** The timestamp when compilation status was recorded. */
+  time: string;
```

Comment:
>string [](http://example.com/codeflow?start=8&length=6)

consider modeling as utcDateTime or at least clearly documenting the timestamp format

Reply:
UTC isn't a format its just a time zone. Whats the format? RFC something? ISO something? Other?

Reply:
Updated them